test_requirements.txt

```diff
@@ -0,0 +1,8 @@
+flake8==2.6.2
+mccabe==0.5.0
+pika==0.10.0
+pycodestyle==2.0.0
+pyflakes==1.2.3
+requests==2.10.0
```

Comment:
Aren't pika, requests, and structlog app requirements? If so, they should be in a requirements.txt